tests/test_layout_objects.py

```diff
@@ -440,30 +440,13 @@ def test_tab_and_tab_holder(self):
                     "one",
                     "first_name",
                     css_id="custom-name",
-                    css_class="first-tab-class active",
+                    css_class="first-tab-class",
                 ),
                 Tab("two", "password1", "password2"),
+                css_class="tab-holder-class",
```

Comment:
I didn't see test coverage for this change. I've changed the style of this test to test against the whole form which is saved as a separate file.

Reply:
It there was in line 453, but you are right, the new style for test is better 👌